packages/snjs/lib/services/api/session_manager.ts

```diff
@@ -640,6 +640,7 @@ export class SNSessionManager extends PureService<SessionEvent> {
     const user = data.user;
     this.user = user;
     await this.storageService.setValue(StorageKey.User, user);
+    this.apiService.setHost(this.apiService.getHost());
```

Comment:
Why is this needed?

Reply:
See here https://app.asana.com/0/1200980603427013/1201146018036636